src/Service/Monei/MoneiService.php

```
@@ -119,7 +120,12 @@ public function getPaymentMethodsResponse()
 
             return $moneiAccountInformation;
         } catch (\Exception $e) {
-            \PrestaShopLogger::addLog('MONEI - getPaymentMethodsResponse - Error: ' . $e->getMessage(), \Monei::getLogLevel('warning'));
+            $errorMessage = $this->extractErrorMessage($e);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$errorMessage'. (undefined)

(UnusedLocalVariable)